app/marketing/mails.py

```diff
@@ -1090,15 +1090,19 @@ def setup_lang(to_email):
 
 
 def new_bounty_request(model):
-    to_email = 'vivek.singh@consensys.net'
+    to_email = model.github_org_email if model.github_org_email != '' else 'vivek.singh@consensys.net'
     from_email = model.requested_by.email or settings.SERVER_EMAIL
     cur_language = translation.get_language()
     try:
         setup_lang(to_email)
         subject = _("New Bounty Request")
-        body_str = _("New Bounty Request from")
-        body = f"{body_str} {model.requested_by}: "\
-            f"{settings.BASE_URL}_administrationbounty_requests/bountyrequest/{model.pk}/change"
+        body_str = _("New Gitcoin bounty funding request from")
+        body = f"{body_str} {model.requested_by.github_url}: \n\n"\
+               f"Link to the issue: {model.github_url}. \nThe user proposes funding of {model.amount} (USD) "\
+               f"and has attached the following comment along with the funding request:\n \"{model.comment}\".\n\n "\
+               f"Should you wish to fund the aforementioned issue, you can follow this URL:\n\n "\
+               f"https://gitcoin.co/bounty/new?invite={model.requested_by}&source={model.github_url}"
```

Comment:
is sending the username but need to be the user ID

Reply:
Can you clarify where the user id is? Is it in the User model pk?

Reply:
Is the profile id when you get into something like http://localhost:8000/bounty/new?invite=13 will open the bounty fund form with the invite filled with the user profile id 13